src/index2.md

```diff
@@ -99,8 +99,12 @@ layout: home2
         <p class="Home2-testimonial-profileInfo-role">Software Eng. Lead</p>
       </div>
       <div class="Home2-testimonial-profileSocial">
-        <a href="https://twitter.com/theotherelliott" rel="noopener noreferrer" target="_blank"  class="Home2-testimonial-profileSocial-link" >{% svg assets/svg/social-twitter.svg %}</a>
-        <a href="http://engblog.yext.com/author/telliott" rel="noopener noreferrer" target="_blank" class="Home2-testimonial-profileSocial-link">{% svg assets/svg/social-web.svg %}</a>
+        <a href="https://twitter.com/theotherelliott" rel="noopener noreferrer" target="_blank"  class="Home2-testimonial-profileSocial-link" >
```

Comment:
I re-used this Twitter icon for the footer. So instead of baking the stroke color in the svg, we're going to apply it via CSS>

A bit worried this will cause a conflict with the PR for interactive testimonials. But hopefully it'll be an easy merge.